packages/usa-header/src/index.js

```diff
@@ -158,19 +160,41 @@ const handleEscape = (event) => {
   focusNavButton(event);
 };
 
+/**
+ * Takes clicked link and current url, removes any hash from the link, and compares them.
+ * If link destination is determined to be the same page; return true.
+ * If link destination is determined to be on a separate page; return false.
+ *
+ * @param {HTMLAnchorElement} targetLink - Target link clicked in header.
+ * @returns {boolean}
+ */
+const isChildSection = (targetLink) => {
+  const baseURL = CURRENT_PAGE.split("#");
+  const linkDestination = targetLink.href;
+  const linkBaseURL = linkDestination.split("#");
+  if (baseURL[0] === linkBaseURL[0]) {
+    return true;
+  }
+
+  return false;
+};
+
 navigation = behavior(
   {
     [CLICK]: {
       [NAV_CONTROL]() {
-        // If another nav is open, close it
-        if (navActive !== this) {
-          hideActiveNavDropdown();
-        }
-        // store a reference to the last clicked nav link element, so we
-        // can hide the dropdown if another element on the page is clicked
-        if (!navActive) {
-          navActive = this;
-          toggle(navActive, true);
+        // Prevent actions for mobile nav so accordions do not close when their child links are clicked.
+        if (!isActive()) {
+          // If another nav is open, close it
+          if (navActive !== this) {
+            hideActiveNavDropdown();
+          }
+          // store a reference to the last clicked nav link element, so we
+          // can hide the dropdown if another element on the page is clicked
+          if (!navActive) {
+            navActive = this;
+            toggle(navActive, true);
+          }
         }
```

Comment:
I was hoping to refactor this into a function, but doing so seemed to break functionality.